schema.md

```diff
@@ -0,0 +1,555 @@
+
+This key defines the types of public administration which is expected
+to use this software. Public software is often specific in scope, because
+there is a large set of tasks that are specific to each type of
+administration. For instance, many softwares that are used in schools
+are probably not useful in hospitals.
+
+The list of allowed values is defined in [pa-types.md](pa-types.md),
+and can evolve at any time, separately from the version of this
+specification.
+
+### Key `meta/category`
+
+* Type: enumerated string
+* Presence: optional
+* Values: see [sw-category.md](sw-category.md)
+* Example: "it-anagrafe"
+
+This key defines the "category" of software, which is the market area
+for the software. You can select the category that is more similar to
+the software; it doesn't have to be an exact match, but it can still
+be very useful for building catalogs of software and evaluate
+alternatives.
+
+The list of allowed values is defined in [sw-category.md](sw-category.md),
+and can evolve at any time, separately from the version of this
+specification.
+
+### Key `meta/tags`
+
+* Type: array of strings
+* Presence: optional
+* Example: "city", "accounting", "hr", "employee", "public"
+
+A list of words that can be used to describe the software and can
+help building catalogs of open software.
+
+Each tag must be in Unicode lowercase, and should not contain
+any Unicode whitespace character. The suggested character to
+separate multiple word is `-` (single dash).
+
+
+### Key `meta/used-by`
+
+* Type: array of strings
+* Presence: optional
+* Example: "City of Amsterdam"
+
+A list of public administrations that are currently using this software.
+
+Parsers are encouraged to enhance this list also with other information
+that can obtain independently; for instance, a fork of a software,
+owned by an administration, could be used as a signal of usage of the
+software.
+
+
+
+
+### Section `dependencies`
+
+This section provides an overview on the system-level dependencies
+required to install and use this software.
+
+**NOTE:** do not list dependencies at the source code level (eg: software
+libraries being used), and focus only on runtime and/or system-level
+dependencies that must be installed and maintained separately. For
+instance, a database is a good example of such dependencies.
+
+### Key `section/open`
+
+* Type: array of strings
+* Presence: optional
+* Example: "MariaDB 10.2"
+
+This key contains a list of runtime dependencies that are distributed
+under an open-source license.
+
+Each string is free form, max 50 characters; feel free to add version
+numbers and/or short comments.
+
+### Key `section/proprietary`
+
+* Type: array of strings
+* Presence: optional
+* Example: "IBM SoftLayer 4.5"
+
+This key contains a list of runtime dependencies that are distributed
+under a proprietary license.
+
+Each string is free form, max 50 characters; feel free to add version
+numbers and/or short comments.
+
+### Key `section/hardware`
```

Comment:
I think you mean `dependencies/hardware`

Reply:
Done